feature/my/src/main/java/com/hankki/feature/my/MyRoute.kt

```diff
@@ -1,28 +1,160 @@
 package com.hankki.feature.my
 
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
 import androidx.compose.foundation.layout.Column
 import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
 import androidx.compose.foundation.layout.padding
-import androidx.compose.material3.Button
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.foundation.verticalScroll
 import androidx.compose.material3.Text
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.hilt.navigation.compose.hiltViewModel
+import androidx.lifecycle.compose.collectAsStateWithLifecycle
+import coil.compose.AsyncImage
+import com.hankki.core.common.extension.noRippleClickable
+import com.hankki.core.designsystem.theme.Gray400
+import com.hankki.core.designsystem.theme.Gray900
+import com.hankki.core.designsystem.theme.HankkiTheme
+import com.hankki.core.designsystem.theme.HankkijogboTheme
+import com.hankki.core.designsystem.theme.Red
+import com.hankki.core.designsystem.theme.White
+import com.hankki.domain.my.entity.UserInfoEntity
+import com.hankki.feature.my.component.ButtonWithArrowIcon
+import com.hankki.feature.my.component.ButtonWithImageAndBorder
+import com.hankki.feature.my.component.MyTitle
 
 @Composable
-fun MyRoute(paddingValues: PaddingValues,  navigateToDummy: () -> Unit) {
-    MyScreen(paddingValues, navigateToDummy = navigateToDummy)
+fun MyRoute(
+    paddingValues: PaddingValues,
+    navigateToDummy: () -> Unit,
+    myViewModel: MyViewModel = hiltViewModel()
+) {
+    val myState = myViewModel.myState.collectAsStateWithLifecycle().value
```

Comment:
by를 사용한다면 .value로 접근하지 않아도 돼요
```suggestion
    val myState by myViewModel.myState.collectAsStateWithLifecycle()
```